### ISSUE TITLE
The orthogonal_loss is not used in the training process

### ISSUE THREAD

**Issue by ZYBOBO** [hqh0728/TimeBase#2]

**Comment by h505023992** [comment]
During training, we incorporate an **orthogonal loss**, and we found that it has a significant impact on more sensitive datasets such as **ETTh1** and **ETTh2**. The loss weight is set to **0.16**, as specified in `scripts/etth1.sh` at **line 41**:

```bash
--orthogonal_weight 0.16
```

We hope this clarifies the training setup. @ZYBOBO
